### DISCUSSION TITLE
Using my own mesh

### DISCUSSION THREAD

**Discussion by ayt1998** [mdolab/dafoam#264] [General]
Hello everyone,
To test DAFoam，I use my own NACA0012 airfoil mesh created by pointwise and tested with OpenFOAM v-1812.
Other conditions are the same as tutorial NACA0012_incompressible. Maximum iteration is also 50.
But my opt_IPOPT.txt show that the target function Cd is not changing less and less.
Why does the official tutorial NACA0012_incompressible stops converging after 17 steps, while my example sometimes gets bigger and sometimes it gets smaller during 50 steps.

Thanks for your advice.

Best regards,
Macro 

[opt_IPOPT.txt](https://github.com/mdolab/dafoam/files/8077927/opt_IPOPT.txt)

**Comment by friedenhe** [comment]
Yeah, this looks strange. The optimization had good progress and then suddenly crashed. I suspect it is related to the IPOPT optimizer (you may need to tweak some parameters to make it work). Can you try SLSQP and rerun it?

**Reply by ayt1998** [reply]
Thank you very much for your answer, slsqp runs perfectly.

**Reply by usmanshahid90** [reply]
Can you please guide me regarding how to use own mesh? In DAFOAM, we need to define symmetry planes. Whereas in ICEM or Pointwise, we mske 2D mesh. When the 2D mesh is converted into openfoam mesh, it creates depth but fornt and back plane has one name rather than two seprate names as symmetry 1 and symmetry 2. How to handle this problem?

please guide.
Thanks

**Reply by uttamcadambi07** [reply]
Okay I am doing a similar thing. I used FLUENT to generate my mesh in *.msh format and used fluentMeshToFoam. This gave me 4 patches - inlet, outlet, wall and sides (the sides being front and back). I set them to be symmetry collectively and it works for me. Why do you want to split it into front and back if you are setting the same boundary condition? Just curious.

**Reply by usmanshahid90** [reply]
Okay then in the runscript file, there exist symmetry planes. What to define for them?

**Reply by uttamcadambi07** [reply]
Check my answer on the other post #383 (redirecting it to the relevant issue so that people can find it there)
